pkgs/development/libraries/portaudio/default.nix

```diff
@@ -5,8 +5,8 @@ stdenv.mkDerivation {
   name = "portaudio-190600-20161030";
 
   src = fetchurl {
-    url = "http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz";
-    sha256 = "04qmin6nj144b8qb9kkd9a52xfvm0qdgm8bg8jbl7s3frmyiv8pm";
+    url = "http://files.portaudio.com/archives/pa_stable_v190700_20210406.tgz";
```

Comment:
```suggestion
    url = "http://files.portaudio.com/archives/pa_stable_v${version}.tgz";
```

Reply:
Thanks for spotting the incorrect version!